src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1520,17 +1527,20 @@ pre.rust {
 #settings-menu, #help-button {
 	margin-left: 4px;
 	outline: none;
+	background-color: var(--search-elements-background-color);
```

Comment:
I think `--search-elements-background-color` is a misleading name for this variable. The settings menu button and the help button are only related to the search button by location, not by semantics. Perhaps this should be `--button-background-color`?

Reply:
They are in the "search-nav" so I thought it made sense but `--button-background-color` sounds better indeed.